apps/database/orm-config.ts

```diff
@@ -0,0 +1,19 @@
+import { PostgresConnectionOptions } from "typeorm/driver/postgres/PostgresConnectionOptions"
+
+const ormConfig: PostgresConnectionOptions = {
+  type: "postgres",
+  host: process.env.DATABASE_HOST || "localhost",
+  port: parseInt(process.env.DATABASE_PORT, 10) || 5433,
+  username: process.env.DATABASE_USERNAME || "postgres",
+  password: process.env.DATABASE_PASSWORD || "postgres",
+  database: process.env.DATABASE_NAME || "stator",
+  synchronize: !!process.env.DATABASE_SYNCHRONIZE || true,
```

Comment:
this is never going to work. if you set it to false, it will fallback to true. you should use the `??` operator instead

Reply:
Yes, good catch!